content/components/inline-message.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Inline Message
+description: InlineMessage is used to inform the user about the result of an action within the content.
+reactId: 
+railsIds:
+figmaId: inline_message
+---
+
+import {Box} from '@primer/react'
+
+<img
+  width="960"
+  alt="Image displaying each variation of InlineMessage, from top: critical, warning, success, unavailable."
+  src="https://github.com/primer/primitives/assets/813754/f9daa010-532a-42e2-b884-45ee6263e17c"
+/>
+
+
+## Usage
+
+InlineMessages inform users about the result of an action they took. In contrast to a [Banner](./banner) an InlineMessage can be very close the where the action has taken place.
+For example below an input field, next to a button or within a table.
+
+## Anatomy
+<img
+  width="960"
+  alt="Anatomy of an InlineMessage, including the icon and the message"
```

Comment:
```suggestion
  alt="A labeled diagram of the inline message component. The labeled parts are an icon and a message."
```

Reply:
Alternatively, you could make this `role="presentation"` if you add text that explains each labelled part of the anatomy diagram. See the [tree view](https://primer.style/components/tree-view) for an example.